test-cases/longevity/longevity-100gb-4h.yaml

```diff
@@ -12,7 +12,7 @@ n_monitor_nodes: 1
 
 instance_type_db: 'i3.4xlarge'
 
-nemesis_class_name: 'ChaosMonkey'
+nemesis_class_name: 'SisyphusMonkey'
```

Comment:
is it on purpose it doesn't have a `nemesis_seed` set?

Reply:
No, I'll fix it.

Reply:
fixed